registrations/api.py

```diff
@@ -304,22 +296,24 @@ def filter_queryset(self, queryset):
                     or registration.registration_user_accesses.filter(
                         email=user.email
                     ).exists()
-                    or registration.publisher in admin_organizations
+                    or registration.publisher in registration_admin_orgs
                 ):
                     raise DRFPermissionDenied(
                         _(
-                            "Only the admins of the organization {organization} have access rights."
-                        ).format(organization=registration.publisher)
+                            f"Only the admins of the organization {registration.publisher} "
```

Comment:
I seem to remember that you cannot mix `gettext` and f-strings. I think the f-string value would be evaluated before `gettext` is run so the sting to translate would already contain the organization name in this case.

Reply:
Good point and definitely applies here -> I changed this back to how it was.